### PR TITLE
(PC-10294) Booking Details, retravailler le QR code pour le Web

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-10294

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [ ] Written **documentation on Notion** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*Web - [Chrome] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
| image
 |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

**Review by BenStirrup — APPROVED** [review]

File: src/features/bookings/components/ThreeShapesTicket.web.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { TicketHeader } from 'ui/svg/TicketHeader'
+import { ColorsEnum, getSpacing } from 'ui/theme'
+
+type Props = PropsWithChildren<{
+  width?: number
+}>
+
+const TICKET_MAX_WIDTH = 300
+
+export function ThreeShapesTicket(props: Props) {
```
Comment (emmajmn) [inline src/features/bookings/components/ThreeShapesTicket.web.tsx:15]: je ne comprends pas bien quand ce composant est utilisé ?

Comment (kopax-polyconseil) [reply src/features/bookings/components/ThreeShapesTicket.web.tsx:15]: Ce ticket sert à afficher le QR code, il est utilisée en place du `ThreeShapesTicket.tsx` native, dans `BookingDetails.tsx`

Comment (emmajmn) [reply src/features/bookings/components/ThreeShapesTicket.web.tsx:15]: ok cool ! et du coup BookingDetails arrive a faire le choix lui même entre les composants web et native en fonction de la platform ou il faut le préciser dans le code ?

Comment (kopax-polyconseil) [reply src/features/bookings/components/ThreeShapesTicket.web.tsx:15]: Via la précédence dans les extensions importé par webpack: https://github.com/pass-culture/pass-culture-app-native/blob/master/web/config/paths.js#L23-L35

**Review by emmajmn — APPROVED** [review]

File: src/features/bookings/components/ThreeShapesTicket.web.tsx (hunk 1 above)
Comment (emmajmn) [reply src/features/bookings/components/ThreeShapesTicket.web.tsx:15]: top c'est clair ! merci pour les explications !

**Pull request merged**